core/client/app/controllers/settings.js

```diff
@@ -1,32 +0,0 @@
-import Ember from 'ember';
-var SettingsController = Ember.Controller.extend({
-
-    needs: ['feature'],
-
-    showGeneral: Ember.computed('session.user.name', function () {
```

Comment:
It hasn't.  The nav bar shows links to everything right now.  Assuming it's reimplemented, it won't be in the settings controller.